docs/source/index.rst

```diff
@@ -1,6 +1,88 @@
-========================
-Scylla Monitoring Stack
-========================
+.. title:: Scylla Monitoring Stack
+
+.. raw:: html
+
+  <div class="landing landing--floating">
+
+
+.. hero-box::
```

Comment:
The hero box does not look visually appealing due to the amount of text:

![image](https://user-images.githubusercontent.com/87854155/135760877-c0acfa12-26b5-435f-beb0-e39a850d75ce.png)

Instead, you could try to make the description shorter as follows:

**Option A) Keep a short description (recommended)**

![image](https://user-images.githubusercontent.com/87854155/135761098-1ac9123a-74b2-40b0-961b-286f54988efa.png)


**Option B)  Remove the advanced description, which is already available here https://monitoring.docs.scylladb.com/master/intro.html**

![image](https://user-images.githubusercontent.com/87854155/135760918-d527a40b-81d5-47bb-a0bd-886f878af3ec.png)

**Option C) Remove where to find old versions of the docs, which is already defined in https://monitoring.docs.scylladb.com/master/intro.html**

![image](https://user-images.githubusercontent.com/87854155/135760980-ffa04fb7-1807-446d-b129-809aec98ed69.png)